src/app/Services/UserService.php

```diff
@@ -45,12 +45,14 @@ public function getUserHomePosts(): array
         $user = User::with('followings')
             ->find(auth()->id());
 
-        $posts = Post::with('user')
+        $posts = Post::with('originalPost', 'user')
             ->whereIn('user_id', $user->followings->pluck('id')->push($user->id))
             ->latest()
             ->paginate(4);
 
-        return ['posts' => $posts];
+        $originalPost = null;
```

Comment:
Why declare a variable for this one? You can directly pass it in the return's array value.